torchsde/_core/misc.py

```diff
@@ -152,9 +152,23 @@ def batch_mvp(m, v):
     return mvp
 
 
-def grad(inputs, **kwargs):
+def grad(outputs, inputs, grad_outputs=None, **kwargs):
```

Comment:
What's the reason for adding `grad_outputs` as an argument? The default in `autograd.grad` is already `None`, so this doesn't change anything.

Reply:
This is mostly to match up with `grad_inputs` in jvp. But I agree this isn't necessary.